### PR TITLE
feat: integrate aura

### LINKED ISSUE #376: Integrate Aura Module

### PR DESCRIPTION
**Pull request by johnletey**
Closes #376

### CODE REVIEW

File: app/upgrades/krypton/upgrade.go [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+					Aliases:  []string{"attousdy"},
+				},
+				{
+					Denom:    "usdy",
+					Exponent: 18,
+				},
+			},
+			Base:    "ausdy",
+			Display: "usdy",
+			Name:    "Ondo US Dollar Yield",
```
Comment (boojamya) [inline app/upgrades/krypton/upgrade.go:42]: Confirming that the name should be same as description?

**Review by boojamya — APPROVED** [review]
Checking on a few confirmations but looks good!

File: app/upgrades/krypton/upgrade.go (hunk 1 above)
Comment (johnletey) [reply app/upgrades/krypton/upgrade.go:42]: Yes, confirmed this with Ondo in Slack!

File: go.mod [hunk 2]
```diff
@@ -154,6 +155,9 @@ replace (
 	// use cosmos compatible ChainSafe/go-schnorrkel
 	github.com/ChainSafe/go-schnorrkel => github.com/ChainSafe/go-schnorrkel v0.0.0-20200405005733-88cbf1b4c40d
 
+	// use send restriction compatible cosmos/cosmos-sdk
+	github.com/cosmos/cosmos-sdk => github.com/noble-assets/cosmos-sdk v0.45.17-0.20240525075530-8fac7fa77ac6
```
[the remark this answers is not in the document]
Comment (johnletey) [reply go.mod]: Correct, this is simply a minimal backport of send restrictions from v0.50 to v0.45 🫡

**Review by g-luca — DISMISSED** [review]

**Review by g-luca — DISMISSED** [review]

**Review by boojamya — APPROVED** [review]

**Review by g-luca — APPROVED** [review]

**Pull request merged**